statsmodels/tsa/vector_ar/irf.py

```diff
@@ -394,8 +394,8 @@ def err_band_sz2(self, orth=False, svar=False, repl=1000, signif=0.05,
         periods = self.periods
         irfs = self._choose_irfs(orth, svar)
         neqs = self.neqs
-        irf_resim = model.irf_resim(orth=orth, repl=repl, T=periods, seed=seed,
-                                   burn=100)
+        irf_resim = model.irf_resim(orth=orth, repl=repl, horizon=periods, seed=seed,
+                                    burn=100)
```

Comment:
Should the `burn` kwarg in the method be passed instead of hard-coding 100?

Reply:
ditto below

Reply:
Saved for irf.py cleanup